### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [dnschneid/crouton#4737] [General]
I need help Makeing crouton 2.0 I am not good at shell or chroot more -DeviateBacon3

**Comment by TomTravis** [comment]
z@localhost:~$ df
Filesystem                                             1K-blocks     Used        Available Use% Mounted on
/dev/mmcblk0p1                                          51392960 18251996         30497920  38% /
shmfs                                                    1989620    46880          1942740   3% /dev/shm
tmpfs                                                    1989620    13840          1975780   1% /tmp
tmpfs                                                     397924        8           397916   1% /run
tmpfs                                                       5120        0             5120   0% /run/lock
tmpfs                                                    1989620      680          1988940   1% /var/host/dbus
/dev/mapper/encstateful                                 15069136   503920         14548832   4% /var/host/timezone
/dev/root                                                2616632  1687244           929388  65% /usr/lib/modules/5.10.101-14962-gede4ccb2bf53
media                                                    1989620        0          1989620   0% /var/host/media
fuse:drivefs://B9779DB6D9423DA12B9FE30CF61B173F 1125899906842624 14209088 1125899892633536   1% /var/host/media/fuse/drivefs-8bb900ac95d4f452162c028e9e949cb2
/dev/sdb                                               189172872 76597144        102946752  43% /var/host/media/removable/sd200
/dev/sda                                                30192504       24         28640096   1% /var/host/media/removable/deboot
none                                                     1989620        0          1989620   0% /sys/fs/cgroup
none                                                     1989620        4          1989616   1% /sys/fs/selinux
z@localhost:~$ cd /dev
z@localhost:/dev$ ls
ashmem            disk         i2c-0        input         loop6       mem           mmcblk0p6    ptmx    stdout   usbmon0      video0   video2   video30  video8
binder            dm-0         i2c-1        jpeg-enc      loop7       mmcblk0       mmcblk0p7    pts     tpm0     usbmon1      video1   video20  video31  video9
block             dm-1         i2c-12       kmsg          loop8       mmcblk0boot0  mmcblk0p8    random  tpmrm0   usbmon2      video10  video21  video32  vndbinder
bsg               dm-2         i2c-13       kvm           loop9       mmcblk0boot1  mmcblk0p9    rfkill  tty      v4l          video11  video22  video33  vsock
bus               dri          i2c-2        log           mali0       mmcblk0p1     mmcblk0rpmb  rtc     ttyS0    v4l-subdev0  video12  video23  video34  watchdog
camera-internal1  drm_dp_aux0  i2c-3        loop-control  mapper      mmcblk0p10    mtd0         rtc0    ttyS2    v4l-subdev1  video13  video24  video35  watchdog0
char              fd           i2c-4        loop0         media-dec0  mmcblk0p11    mtd0ro       sda     ttyS3    v4l-subdev2  video14  video25  video36  xt_qtaguid
chromeos-low-mem  full         i2c-5        loop1         media0      mmcblk0p12    net          sdb     ttynull  vga_arbiter  video15  video26  video37  zero
console           fuse         i2c-6        loop2         media1      mmcblk0p2     null         shm     udmabuf  vhci         video16  video27  video4   zram0
cpu_dma_latency   gpiochip0    iio:device0  loop3         media2      mmcblk0p3     pmsg0        snd     uhid     vhost-vsock  video17  video28  video5
cros_ec           hwbinder     iio:device1  loop4         media3      mmcblk0p4     port         stderr  uinput   video-dec0   video18  video29  video6
cros_scp          hwrng        image-proc0  loop5         media4      mmcblk0p5     ppp          stdin   urandom  video-enc0   video19  video3   video7
z@localhost:/dev$ cat /etc/os*
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
z@localhost:/dev$ uname -a
Linux localhost 5.10.101-14962-gede4ccb2bf53 #1 SMP PREEMPT Wed Apr 13 19:47:19 PDT 2022 aarch64 aarch64 aarch64 GNU/Linux
z@localhost:/dev$ /dev/root                                                2616632  1687244           929388  65% /usr/lib/modules/5.10.101-14962-gede4ccb2bf53
media                                                    1989620        0          1989620   0% /var/host/media
fuse:drivefs://B9779DB6D9423DA12B9FE30CF61B173F 1125899906842624 14209088 1125899892633536   1% /var/host/media/fuse/drivefs-8bb900ac95d4f452162c028e9e949cb2
/dev/sdb                                               189172872 76597144        102946752  43% /var/host/media/removable/sd200
/dev/sda                                                30192504       24         28640096   1% /var/host/media/removable/deboot
none                                                     1989620        0          1989620   0% /sys/fs/cgroup
none                                                     1989620        4          1989616   1% /sys/fs/selinux
z@localhost:~$ cd /dev
z@localhost:/dev$ ls
ashmem            disk         i2c-0        input         loop6       mem           mmcblk0p6    ptmx    stdout   usbmon0      video0   video2   video30  video8
binder            dm-0         i2c-1        jpeg-enc      loop7       mmcblk0       mmcblk0p7    pts     tpm0     usbmon1      video1   video20  video31  video9
block             dm-1         i2c-12       kmsg          loop8       mmcblk0boot0  mmcblk0p8    random  tpmrm0   usbmon2      video10  video21  video32  vndbinder
bsg               dm-2         i2c-13       kvm           loop9       mmcblk0boot1  mmcblk0p9    rfkill  tty      v4l          video11  video22  video33  vsock
bus               dri          i2c-2        log           mali0       mmcblk0p1     mmcblk0rpmb  rtc     ttyS0    v4l-subdev0  video12  video23  video34  watchdog
camera-internal1  drm_dp_aux0  i2c-3        loop-control  mapper      mmcblk0p10    mtd0         rtc0    ttyS2    v4l-subdev1  video13  video24  video35  watchdog0
char              fd           i2c-4        loop0         media-dec0  mmcblk0p11    mtd0ro       sda     ttyS3    v4l-subdev2  video14  video25  video36  xt_qtaguid
chromeos-low-mem  full         i2c-5        loop1         media0      mmcblk0p12    net          sdb     ttynull  vga_arbiter  video15  video26  video37  zero
console           fuse         i2c-6        loop2         media1      mmcblk0p2     null         shm     udmabuf  vhci         video16  video27  video4   zram0
cpu_dma_latency   gpiochip0    iio:device0  loop3         media2      mmcblk0p3     pmsg0        snd     uhid     vhost-vsock  video17  video28  video5
cros_ec           hwbinder     iio:device1  loop4         media3      mmcblk0p4     port         stderr  uinput   video-dec0   video18  video29  video6
cros_scp          hwrng        image-proc0  loop5         media4      mmcblk0p5     ppp          stdin   urandom  video-enc0   video19  video3   video7
z@localhost:/dev$ cat /etc/os*
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
download debootstrap and read debian chroot, 
it is required you gain experience , systemD is not crouton

**Comment by TomTravis** [comment]
come on over 
you need help join the force
things are changing daily
I hope this helps confused @DeviateBacon3 @myelin @jcs 
i need systemd interest, simple start up raw debootstrap SUSE supported
